src/main/webapp/script.js

```diff
@@ -129,4 +132,20 @@ function getZipCode() {
   xhttp.send();
 
   setTimeout(fetchList, 4000);
+}
+
+function initMap(mapLocation) {
```

Comment:
Let's move the map functions into their own .js file

Reply:
Agreed.

Reply:
It has been moved!